src/assets/diagnostics.yaml

```diff
@@ -0,0 +1,28 @@
+diagnostics:
+  # Format: output from `date -u -Iminutes`
+  # or in JS:  new Date().toISOString()
+  # these differ, but are both parsable
+  last_update: 2022-08-15T16:00+00:00
+  categories:
+    - title: Utilities
+      checks:
+        - id: RD_BIN_IN_BASH_PATH
+          documentation: path#rd_bin_bash
+          description: "The ~/.rd/bin directory has not been added to the PATH, so command-line utilities are not configured in your bash shell."
+          mute: false
```

Comment:
> The user can "mute" specific checks, so they will not generate notifications and not be shown by default.

Initially, nothing is muted. But we'll need a separate store, similar to `settings.json` that tracks the actual user's preferences. Not worked out yet.

Reply:
Yes, the user can mute things, but it would never be relevant for the hard-coded initial state. (We'll need to store the mute-ness elsewhere somehow, but we can't write into the bundled YAML file.)

Reply:
done